common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -194,24 +194,27 @@ colored_emblem_texture_lists = {
 			
```

Comment:
I moved Druidic and Ancient, but since Elunic is shared by Highborne and Nightborne, `charge` might be a better balance. Moreover roughly a third of the `pagan_single_charge_list` is already included in `druidic_coa_gfx`, which Night Elves and Dryads use, so `pagan_single_charge_list` is redundant. `druidic_coa_gfx`

Reply:
It works so due to the vanilla. I don't like it, but it's how it's.

You may change how it works by removing `pagan_single_charge_list` and replacing it with `charge`, but it's up to you and should be done on a separate branch. Thus `charge` will work like a united CoA list.

Reply:
> EVERY CoA list should be in charge.
Pagan-faith-related cultures should be in pagan_single_charge_list: ancients, night elves, ancients.

You miss:
Elunic CoAs in pagan_single_charge_list.
Druidic and ancient CoAs in charge.

I tried this but the problem is that when I add the above to their corresponding lists, elunic CoAs stop appearing in my tests. It's almost like `druidic_coa_gfx` and the base `charge` list emblems completely override `elunic_coa_gfx`. I have tested many times and not a single elunic coa appeared for the affected cultures (kaldorei, dryad - any culture that uses both `elunic_coa_gfx` and `druidic_coa_gfx`) so I am certain it's not a matter of weighting.

Reply:
> I tried this but the problem is that when I add the above to their corresponding lists, elunic CoAs stop appearing in my tests. It's almost like `druidic_coa_gfx` and the base `charge` list emblems completely override `elunic_coa_gfx`. I have tested many times and not a single elunic coa appeared for the affected cultures (kaldorei, dryad - any culture that uses both `elunic_coa_gfx` and `druidic_coa_gfx`) so I am certain it's not a matter of weighting.

You may push these changes to the branch, I'll test it too. It may be due to another reason.

Reply:
Pushed new commit for testing:
9c6039a9f is with all cultural coas in charge and pagan single charge. (Elunic not displaying at all)
741402b0b is previous commit with elunic, druidic and ancient missing. (Elunic displays properly here)

Reply:
Resolved with c311864